packages/nitro-protocol/src/contract/transaction-creators/nitro-adjudicator.ts

```diff
@@ -1,17 +1,18 @@
 // @ts-ignore
 import {providers} from 'ethers';
-import {utils} from 'ethers';
 import NitroAdjudicatorArtifact from '../../../build/contracts/NitroAdjudicator.json';
 import {getChannelId} from '../channel';
 import {encodeOutcome, Outcome} from '../outcome';
 import {getFixedPart, hashAppPart, hashState, State} from '../state';
+import {Signature, Interface} from 'ethers/utils';
 
 // TODO: Currently we are setting some arbitrary gas limit
 // To avoid issues with Ganache sendTransaction and parsing BN.js
 // If we don't set a gas limit some transactions will fail
 const GAS_LIMIT = 3000000;
 
-const NitroAdjudicatorContractInterface = new utils.Interface(NitroAdjudicatorArtifact.abi);
+// @ts-ignore
+const NitroAdjudicatorContractInterface = new Interface(NitroAdjudicatorArtifact.abi);
```

Comment:
For some reason this is throwing a build error, not sure why.

Reply:
Are the contract artifacts built?

Reply:
I see no build error. I do see linting errors